src/DependencyInjection/Configuration.php

```diff
@@ -214,4 +215,23 @@ private function createSmsConfiguration(ArrayNodeDefinition $root)
                 ->end()
             ->end();
     }
+
+    private function createLocaleCookieConfiguration(ArrayNodeDefinition $root)
+    {
+        $root
+            ->children()
+                ->arrayNode('locale_cookie')
+                    ->info('Cookie settings for locale cookie')
+                    ->isRequired()
+                    ->children()
+                        ->scalarNode('name')->end()
+                        ->scalarNode('domain')->end()
+                        ->integerNode('expire')->end()
+                        ->scalarNode('path')->end()
+                        ->booleanNode('secure')->end()
+                        ->booleanNode('httpOnly')->end()
```

Comment:
Hmm, we do provide sane values in the application's configuration (eg. `secure: true`), while keeping all options open in the bundle.

> Though I don't really see any security reasons for this write-only cookie for interop to withhold configuration.

Very well. No need to put more time into this.